Changelogs for Digital Twin Manager Plugins 3.2.0 (#3679)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1022-5-0-new-data-points-plugin-to-manage-data-points-on-assets.md

---

> date: ""

Reviewer: ```suggestion
date: "04-09-2025"
```

> date: "04-09-2025"

Reviewer: ```suggestion
date: "2025-09-04"
```

Reviewer 2: I think we should not pass date as per Beate earlier comment. Please cross check and pass if it is only required.

title: "New Data points plugin to manage data points on assets"
product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2279"
version: "1022.5.0"
---

> A new plugin, Data Points is added to dtm-plugins extension. This allows users to view, link and manage data points on assets.

Reviewer 3: ```suggestion
The dtm-plugins extension introduces a Data Points plugin that enables linking and management of device data points on asset instances. It provides a visual interface for linking device measurements to asset contexts, supports data point templates, and allows adding custom data points to link expected data points before the data arrives—enriching asset metadata through structured associations.
```
